front_end/Credits.html

```diff
@@ -36,6 +36,17 @@ <h2 id="text-credits">Credits</h2>
         <p class="leftcredits-text">Vladyslav Shatskyi</p>
       </div>
     </div>
+
+    <div class="github-link">
+      <a href="https://github.com/DIT112-V21/group-14" target="_blank">
+        <img src="img/GitHub-Mark-120px-plus.png" alt="GitHub Logo">
+      </a>
+    </div>
+
+    <div class="grp">
```

Comment:
The div class `grp` doesn't seem like it is being used in the style sheet. So from what I can see it doesn't seem to have a purpose.

```suggestion
    <div>
```